man/roxygen/rdx_meta.R

```diff
@@ -14,7 +16,8 @@ list(
     msg = "a console message to be printed, e.g. `\"Attention\"` or for longer messages use `paste(\"Line 1\", \"Line 2\")`",
     msg_type = "`\"none\"`, `\"message\"`, `\"warning\"`, `\"error\"`",
     source_list = "a list of source objects, e.g., `list(pd, death)`",
+    var_list = "list of variables created by `exprs()`, e.g., `exprs(USUBJID, VISIT)`",
     var = "a variable",
-    var_list = "list of variables created by `exprs()`, e.g., `exprs(USUBJID, VISIT)`"
+    event = "an `event()` or `event_joined()` object"
```

Comment:
minor comment but easier to maintain if this is alphabetised

Reply:
done